registers/trillian_client/client.go

```diff
@@ -52,17 +53,23 @@ func (t *trillianClient) Scan(logID int64, s LogScanner) error {
 		log.Fatalf("Can't get log root: %v", err)
 	}
 
-	ts := lr.SignedLogRoot.TreeSize
-	for n := int64(0); n < ts; {
-		g := &trillian.GetLeavesByRangeRequest{LogId: logID, StartIndex: n, Count: chunk}
+	var root types.LogRootV1
+	// TODO(Martin2112): Verify root signature.
+	if err := root.UnmarshalBinary(lr.SignedLogRoot.LogRoot); err != nil {
+		return fmt.Errorf("Root failed to unmarshal: %v", err)
```

Comment:
nit: start with lowercase letter

Reply:
Yeah there's a bunch of them. Going to fix separately.